### PR TITLE
Sk ac navbar

### LINKED ISSUE #29: Navbar
### Description

Make a Navbar component. Do NOT use the chakra Tabs component for this, because it kinda sucks. Please ask Varun for help, because the component must also be responsive. You will have to add a media query to switch to a hamburger menu. Decide on a good breakpoint and make it a constant. 

### Acceptance Criteria

Do it look nice? Make sure all links look presentable and are clickable at any screen width. This is super super important!!

### PR DESCRIPTION
**Pull request by al-chang**
### ℹ️ Issue

Closes #29 

### 📝 Description

- Add styling for desktop and mobile nav bar views
- Create hamburger menu with accessibility support
- Add OutstandingLife logo

### ✔️ Verification

Desktop view:
Screen Shot 2022-05-16 at 7 51 14 PM

Mobile view: 
Screen Shot 2022-05-16 at 7 53 38 PM
Screen Shot 2022-05-16 at 7 53 50 PM

### 🏕️ (Optional) Future Work / Notes

- Update logos once we get the right ones

### CODE REVIEW

**Comment by saahilkumar** [comment]
Note: I thought the blue focus color when clicking on the hamburger menu icon was ugly. I wanted to remove it but read that it's bad for accessibility (not sure if that's applicable in this case though). So, I instead changed the focus color to match the color theme of the website. See top right corner:

Screen Shot 2022-05-17 at 12 21 19 AM

**Comment by saahilkumar** [comment]
Removed focus blue outline on the actual NavLinks because they already have the purple underline when selected so I thought the blue outline was useless

**Comment by saahilkumar** [comment]
New logo on mobile:
Screen Shot 2022-05-19 at 1 38 20 PM

New logo on XL:
Screen Shot 2022-05-19 at 1 38 14 PM

File: lib/components/navbar/NavLink.tsx [hunk 1]
```diff
@@ -1,14 +1,29 @@
...
+        height="fit-content"
+        data-cy="nav-bar"
+        color="#4A5568"
+        textDecor={isActive ? 'underline' : 'none'}
+        textUnderlineOffset="3px"
+        textDecorationThickness="3px"
+        textDecorationColor={'var(--purple)'}
+        tabIndex={0}
+        _focus={{ boxShadow: 'none' }}>
+        <chakra.span fontWeight="bold" fontSize="24px">
```
Comment (jessicasu77) [inline lib/components/navbar/NavLink.tsx:23]: just curious what the difference is between using a chakra.span vs a chakra box?

**Review by jessicasu77 — DISMISSED** [review]
Looks good to me!

File: lib/components/navbar/links.ts [hunk 2]
```diff
@@ -1,20 +1,32 @@
...
-const navLinks: NavLinkProps[] = [
+const navLinks: LinkProps[] = [
   {
-    label: 'Contact',
-    path: '/contact',
+    label: 'About Us',
+    path: '/about-us',
+  },
+  {
+    label: 'Membership',
```
Comment (varunthakkar1) [inline lib/components/navbar/links.ts:12]: Don't think we need membership

**Review by varunthakkar1 — CHANGES_REQUESTED** [review]
Please add a cypress test for the links in both desktop and hamburger menu view and remove "membership" from the links. Otherwise looks really good

File: lib/components/navbar/NavLink.tsx (hunk 1 above)
Comment (saahilkumar) [reply lib/components/navbar/NavLink.tsx:23]: Not 100% sure but if I had to guess after reading the docs a little, a chakra.span just renders as `<span>` while a box renders as a `<div>` by default. But, a box can also be a `<span>` if you add `as=span` to it.

**Review by varunthakkar1 — DISMISSED** [review]

File: lib/components/navbar/NavLink.tsx (hunk 1 above)
Comment (al-chang) [reply lib/components/navbar/NavLink.tsx:23]: Yeah there was no particular reason for choosing the `<chakra.span>` over the `<Box as="span">` syntax. Pretty sure they just accomplish the same thing. @jessicasu77

**Pull request merged**